pkg/cmd/ui/ui.go

```diff
@@ -233,11 +235,22 @@ func SelectOptionPushToGit() bool {
 }
 
 // UseDefaultValues allows users to use default values so that they will be prompted with fewer questions in interactive mode
-func UseDefaultValues() bool {
+func UseDefaultValues(defaultFlagValues map[string]interface{}) bool {
+	// Print default values
+	flagValues := "\n\nDefault values (Can be overwritten from the command line):\n"
```

Comment:
Thought about this some more. Let's remove the parentheses:

The default values used for the options, if not overwritten from the command line, are: